### ISSUE TITLE
Browser support arguments should all follow the same order

### ISSUE THREAD

**Issue by mirisuzanne** [Compass/compass#1000]
Currently we have:

``` scss
// -o before -ms
@mixin experimental($property, $value, $moz, $webkit, $o, $ms, $khtml, $official) {}
@mixin experimental-value($property, $value, $moz, $webkit, $o, $ms, $khtml, $official) {}

// -ms before -o
@mixin set-experimental-support($moz, $webkit, $ms, $o, $khtml) {}
@mixin with-only-support-for($moz, $webkit, $ms, $o, $khtml, $legacy-ie) {}
```

**Comment by mirisuzanne** [comment]
Either way we go, this is going to be a painful fix for some people.

**Comment by scottdavis** [comment]
Did this happen?

**Comment by mirisuzanne** [comment]
No.

**Comment by mirisuzanne** [comment]
It looks like our most user-facing mixins take the `-ms, -o` approach, which makes sense to me. I'll start flipping the other mixins to work the same way.

**Comment by mirisuzanne** [comment]
Closed by 707b0a37
